python/hail/representation/genotype.py

```diff
@@ -295,6 +297,7 @@ def one_hot_genotype(self, num_genotypes):
         return jiterable_to_list(from_option(self._jrep.oneHotGenotype(num_genotypes)))
 
     @handle_py4j
+    @typecheck_method(theta=nullable(numeric))
```

Comment:
I think the docs are wrong here. This isn't nullable. Can you make this just numeric and fix the docstring?